src/main/webapp/app/overview/courses.component.ts

```diff
@@ -32,11 +32,17 @@ export class CoursesComponent implements OnInit {
         private teamService: TeamService,
     ) {}
 
+    /**
+     * Life cycle hook called by Angular to indicate that Angular is done creating the component
```

Comment:
Mention the methods being called here and roughly describe what they do, as you have done on the other life cycle hooks where you have added documentation. If the methdos used contain documentation, you can link to them so that you do not repeat yourself.